### ISSUE TITLE
Webpack suddenly compiles to ES6 instead of ES5

### ISSUE THREAD

**Issue by borislemke** [AngularClass/angular2-webpack-starter#740]
- **I'm submitting a ...**
  [x] bug report
  [ ] feature request
  [ ] question about the decisions made in the repository
- **Do you want to request a _feature_ or report a _bug_?**
  _bug_
- **What is the current behavior?**
  My project suddenly compiles into ES6 instead of ES5. I haven't changed anything besides adding a less loader into config/webpack.common.js. Suddenly, I get these "Unexpected token import" errors when trying to run my app. When I open the main.bundle.js files I see ES6 syntaxes such as import present in the file. Anyone with similar experience?
- **What is the expected behavior?**
  Compile into ES5
- **What is the motivation / use case for changing the behavior?**
  Cannot run my app, stuck in "Loading..."
- **Please tell us about your environment:**
- Angular version: 2.0.0-rc.3
- Browser: [all]
- **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)
  `async.js:1               Uncaught SyntaxError: Unexpected token import`

**Comment by gdi2290** [comment]
did you change anything in the config files. are you on windows and are you using npm 3?

**Comment by borislemke** [comment]
@gdi2290 Hi Patrick, I did not change anything in the config, however, I managed to get it working again.
After disabling my components one by one, I found out that this particular import causes the error:
`import {Subscription} from "rxjs/Rx";`
Commenting this import resolves the issue. I wanted to do something like `this.subscription.unsubscribe()` on a component's `ngOnDestroy` where `this.subscription`'s type is set to 'Subscription' imported from `rxjs/Rx`.
